### ISSUE TITLE
Bias in The First Layer

### ISSUE THREAD

**Issue by maju3634** [EdoVaira/Iris-Neural-Network#2]
Hello, i am trying to learn your algorithm to classify this iris-data, and the result is good.
But I have a question, is it good to add the bias node to compute the first hidden layer? As i know, we dont need to add the bias node when we want to compute the first hidden layer.

`hidden_output = tf.nn.softmax(tf.add(tf.matmul(x_data, w1),b1))`

but we can change this code to:

`hidden_output = tf.nn.softmax(tf.matmul(x_data, w1))`

Am I right? Hopefully you respond my question, thank you!
